imgui/core.pyx

```diff
@@ -1935,6 +1935,220 @@ def menu_item(
     return clicked, inout_selected
 
 
+def open_popup(str name):
+    """Open a popup window.
+
+    Marks a popup window as open. Popups are closed when user click outside,
+    or activate a pressable item, or :func:`close_current_popup()` is
+    called within a :func:`begin_popup()`/:func:`end_popup()` block.
+    Popup identifiers are relative to the current ID-stack
+    (so :func:`open_popup` and :func:`begin_popup` needs to be at
+    the same level).
+
+    .. visual-example::
+        :title: Simple popup window
+        :height: 100
+        :width: 220
+        :auto_layout:
+
+        imgui.begin("Example: popup window")
```

Comment:
After moving popup modal example to `begin_popup_model()` function docstring we could rewrite it to something simpler like:

        imgui.begin("Example: simple popup")

        if imgui.button("select"):
            imgui.open_popup("select-popup")

        imgui.same_line()

        if imgui.begin_popup_modal("select-popup")[0]:
            print("pop")
            imgui.text("Select one")
            imgui.separator()
            imgui.selectable("One")
            imgui.selectable("Two")
            imgui.selectable("Three")
            imgui.end_popup()

        imgui.end()

That looks like this:
![example_snippet](https://cloud.githubusercontent.com/assets/1258054/23614566/ac96fabc-0283-11e7-8858-484166fede5b.png)